### DISCUSSION TITLE
Error in import

### DISCUSSION THREAD

**Discussion by mohammadrezaghorvei** [pyg-team/pytorch_geometric#4407] [General]
please help me with this error

OSError: /usr/local/lib/python3.7/dist-packages/torch_scatter/_scatter_cuda.so: undefined symbol: _ZNK2at6Tensor6narrowElll

**Comment by rusty1s** [comment]
Can you give me some more context here? How did you try to install `torch-scatter`? What's your local PyTorch and CUDA version? In general, this error message indicates a version mismatch between existing libraries.
